nncf/experimental/torch/fx/quantization/quantize_model.py

```diff
@@ -103,3 +109,46 @@ def quantize_impl(
     quantized_model = _disallow_eval_train(quantized_model)
 
     return quantized_model
+
+
+def compress_weights_impl(
```

Comment:
Could you check that `compress_weigths` from Torch FX backend supports for models with dynamic_shapes? If it does not support, please open an issue.

Reply:
Alright, I will test it out

Reply:
Testing it with some synthetic models(like `ShortTransformer()` from `test_compress_weights.py`), it appears to be compressing, and also I am able to run inference with it using differently shaped inputs.